### PR TITLE
[17.0][OU-ADD] account: Migration to 17.0

### PR DESCRIPTION
**Pull request by pedrobaeza**
Continuation of #4457 

@Tecnativa TT49274

### CODE REVIEW

File: openupgrade_scripts/scripts/account/17.0.1.2/pre-migration.py [hunk 1]
```diff
@@ -0,0 +1,355 @@
...
+    openupgrade.rename_xmlids(env.cr, xmlids_renames)
+
+
+def _convert_account_tax_description(env):
+    openupgrade.rename_columns(env.cr, {"account_tax": [("description", None)]})
+    openupgrade.logged_query(env.cr, "ALTER TABLE account_tax ADD description JSONB")
+    openupgrade.logged_query(
+        env.cr,
+        f"""UPDATE account_tax
+        SET description = {'en_US': {openupgrade.get_legacy_name('description')}}
```
Comment (pedrobaeza) [inline openupgrade_scripts/scripts/account/17.0.1.2/pre-migration.py:201]: This is not working yet.

**Comment by pedrobaeza** [comment]
@hbrunn do you want to work on the rest?

**Comment by hbrunn** [comment]
@pedrobaeza can you make this PR writable for maintainers?

**Comment by pedrobaeza** [comment]
I'm afraid it's something of GitHub itself, as it has been pushed from a organization branch. Feel free to open a new one if you finish it. There's now a CI problem whose roots have been investigated here: https://github.com/Tecnativa/doodba/pull/636

**Comment by legalsylvain** [comment]
Indeed. 

- If pedro creates a branch on his personal space (pedrobaeza) such option are available : 

image

- But if he creates a branch on his organization space (Tecnativa), no option is available.

(tested with my legalsylvain / grap branches)

**Comment by hbrunn** [comment]
I won't be able to add anything today, will have a look if there's anything left over next Monday

**Comment by hbrunn** [comment]
PS: Let's then to PRs from our personal forks to facilitate collaboration in the future

**Comment by pedrobaeza** [comment]
OK, I will continue with this PR during the week.

> I won't be able to add anything today, will have a look if there's anything left over next Monday

Yeah, it was the habit, as in our team we always push everything to Tecnativa to promote that collaboration (as others don't have repo maintainer role).

**Comment by legalsylvain** [comment]
> PS: Let's then to PRs from our personal forks to facilitate collaboration in the future

We can also make a PR on OpenUpgrade repo. (OpenUpgrade:17.0-account) So all the maintainers can push on it. 
Pro : better collaboration. Cons : corporate OCA statistics doesn't include such work. (= how many PRs are done by tecnativa each years.)
Maybe such things could be done only for 2 or 3 PR per revision, for modules that requires huge work (like account, or base, ...)

**Comment by hbrunn** [comment]
good idea about pushing to /oca, will do. Continuing this now

**Comment by hbrunn** [comment]
superseded by https://github.com/OCA/OpenUpgrade/pull/4668

**Pull request closed without merging**